src/wram.asm

```diff
@@ -955,8 +955,8 @@ wTempNonTurnDuelistCardID:: ; ccc4
 wccc5:: ; ccc5
 	ds $1
 
-; may contain 0 or 1 depending on which move was selected
-wSelectedMoveIndex:: ; ccc6
+; *_ATTACK constants for selected attack
```

Comment:
I like the reference to the constant, but I'm not sure if it's clear enough, so I'd probably keep something like the previous comment referencing the constant values.

Reply:
Kept reference to the values.